HARK/utilities.py

```diff
@@ -1101,3 +1105,41 @@ def find_gui():
     if plt.get_backend() == "Agg":
         return False
     return True
+
+
+def benchmark(
+    agent_type, sort_by="tottime", max_print=10, filename="restats", return_output=False
+):
+    """
+    Profiling tool for HARK models. Calling `benchmark` on agents calls the solver for
+    the agents and provides time to solve as well as the top `max_print` function calls
+    in terms of  `sort_by`. Optionally allows for saving a text copy of the profile
+    as well as returning the `Stats` object for further inspection.
+
+    Parameters
+    ----------
+    agent_type: AgentType
+            A HARK AgentType with a solve() method.
+    sort_by: string
```

Comment:
As this is passed directly to pstats.Stats, it would make sense to have a link in the docstring to the relevant python documentation. As is, you really have to look at the code to understand how to use the parameters